docs/vspec2x_arch.md

```diff
@@ -22,6 +19,28 @@ This will result in that e.g. `Vehicle.Cabin.Door` is expanded to the following
 For some exporters expansion can be suppressed by using the `--no_expand` option.
 Then instance information will be represented by other means in the resulting output.
 
+## Deletion / Node removal
+
+Nodes can be removed from the tree by using the `delete` keyword in the overlay.
+This is useful when a signal is not used in a next version of the specification or if you
+simply want to delete a node or branch from the specification, e.g. you are not using a signal of the base
+specification.
+Please note, that the deletion for branches will delete all nodes that are connected to that branch (which is
```

Comment:
Should make it clear that adding the boolean delete element set to true to a branch node causes all sub-nodes to be ignored and thus their delete element is irrelevant.